harness/determined/pytorch/_pytorch_trial.py

```diff
@@ -9,16 +10,18 @@
 import warnings
 from abc import abstractmethod
 from inspect import signature
-from typing import Any, Callable, Dict, Iterator, List, Optional, Type, Union, cast
+from typing import Any, Callable, Dict, Iterator, List, Optional, Tuple, Type, Union, cast
 
 import numpy as np
 import torch
 import torch.distributed as dist
 
 import determined as det
-from determined import layers, pytorch, tensorboard, util, workload
+from determined import core, pytorch, tensorboard, util
+from determined.common import check
```

Comment:
no using the check module in any new code.

Reply:
done